packages/common-react/tsconfig.json

```diff
@@ -29,9 +29,9 @@
     "moduleResolution": "node" /* Specify how TypeScript looks up a file from a given module specifier. */,
     // "baseUrl": "./",                                  /* Specify the base directory to resolve non-relative module names. */
     "paths": {
-      "src/theme": ["../../streamlit/frontend/src/theme"],
```

Comment:
Looks like the `paths` config is no longer needed anyway, and it should simply be commented-out, just like `kernel/tsconfig.json` [here](https://github.com/whitphx/stlite/pull/671/files#diff-618cfccfdc1756d1182cf543744c85f8c1ddfc5a2e5a6002cea4bbf63ce70e9cL31-L35) (sorry I should have updated this file as well in the base branch.)

Reply:
Removed 👍